src/slices/GiveThunk.ts

```diff
@@ -2,7 +2,6 @@ import { ethers } from "ethers";
 import { addresses } from "../constants";
 import { abi as ierc20Abi } from "../abi/IERC20.json";
 import { abi as OlympusGiving } from "../abi/OlympusGiving.json";
-// Delete before mainnet
 import { abi as MockSohm } from "../abi/MockSohm.json";
```

Comment:
Shouldn't this be removed too?

Reply:
It depends on if we want to include a way to drip on testnet. The abi basically determines what methods ethers allows you to call and the standard ierc20Abi or the standard sOHM abis do not have a drip function. So even if it's not called MockSohm we would need an abi for it. I can remove it for now though and comment out the "getTestTokens" function until we come up with a better solution